db/seeds.rb

```diff
@@ -5,3 +5,19 @@
   first_name: 'Test',
   last_name: 'TestLastName'
 )
+
+Book.create(
```

Comment:
Use FactoryBot for the seeds:
```
FactortyBot.create(:book)
FactoryBot.create(:user) 
```

Remember to use Faker in the factory bot

Reply:
We'll user default active record syntax as discussed in Slack